controllers/mysql_combined_test.go

```diff
@@ -53,6 +53,27 @@ func TestMySQLHappyPath(t *testing.T) {
 
 	EnsureInstance(ctx, t, tc, mySQLDBInstance)
 
+	mySQLAdmin := &azurev1alpha1.MySQLServerAdministrator{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "admin",
+			Namespace: "default",
+		},
+		Spec: azurev1alpha1.MySQLServerAdministratorSpec{
+			ResourceGroup:     rgName,
+			Server:            mySQLServerName,
+			AdministratorType: azurev1alpha1.MySQLServerAdministratorTypeActiveDirectory,
+			// The below fields are for a user managed identity which exists in the ASO-CI subscription
```

Comment:
Possibly we need to support creating managed identities via the operator too in which case we could do an e2e test that did that first and then used it here

Reply:
I've asked Akin about this

Reply:
Sounds like we don't need to support managed identity creating at least for now, so this is probably the best path forward at least temporarily.